src/components/popover-menu/popover-menu-button.tsx

```diff
@@ -0,0 +1,21 @@
+import { PopoverButton as HeadlessUiPopoverButton } from "@headlessui/react";
+import React from "react";
+import { usePopoverMenuContext } from "./popover-menu-context";
+import { Button, ButtonProps } from "../button/button";
+
+export interface NavigationPopoverButtonProps extends ButtonProps {
+    children: React.ReactNode;
+    onClick?: () => void;
+}
```

Comment:
I imagine depends on the usage, but maybe we want to have the same styling of the typical buttons? As in primary, danger, minimal, etc?

Reply:
hihi yes, we are doing that already as you can see in the implementation:

```
export interface PopoverMenuButtonProps extends ButtonProps {
    children: React.ReactNode;
    onClick?: () => void;
}

export const PopoverMenuButton = ({ onClick, children, ...restProps }: PopoverMenuButtonProps) => {
    const {
        popoverButton: { setReferenceElement },
    } = usePopoverMenuContext();

    return (
        <HeadlessUiPopoverButton ref={(el) => el && setReferenceElement(el)} onClick={onClick}>
            <Button {...restProps}>{children}</Button>
        </HeadlessUiPopoverButton>
    );
};
```

Reply:
Ah, I missed we're importing the button component here, I was judging bases on storybook because it had no controls for it. Ok, you can resolve this